src/test/components/__snapshots__/TrackMemory.test.js.snap

```diff
@@ -146,6 +129,23 @@ Array [
   Array [
     "fill",
   ],
+  Array [
+    "clearRect",
+    0,
+    0,
+    80,
+    10,
+  ],
+  Array [
+    "scale",
+    1,
+    1,
+  ],
+  Array [
+    "scale",
+    1,
+    1,
+  ],
```

Comment:
I'm not exactly sure of this change. This is the marker canvas for the memory markers inside the memory track. I believe this may be due to the timing of `requestAnimationFrame` calls. I believe this is not important but wanted to call it out.